projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -62,6 +63,32 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
     @Input() public first: boolean;
     @Input() public last: boolean;
     @Input() responsiveBody = true;
+    @Input() preventCollapse = false;
+    @Input() hidePanelHeader = false;
+    @Input() actionButtonTemplates: Array<TemplateRef<any>>;
+    @Input() actionRowJustifyContent: 'space-between' | 'flex-start' | 'flex-end' | 'center' | 'space-around' |
+        'initial' | 'start' | 'end' | 'left' | 'right' | 'revert' | 'inherit' | 'unset'
+
+    thisContext = {
```

Comment:
an intefrace tytpe for this variable should exist and be exported

Reply:
Corrected.